### DISCUSSION TITLE
[New Feature] Add a way to make warheads detonate as soon as they touch a building

### DISCUSSION THREAD

**Discussion by CommanderNour** [Phobos-developers/Phobos#1000] [Ideas and feedback]
### Description:
in the vanilla game inaccurate projectile will detonate their WH as soon as they touch a building, instead of going straight to the center of the building

### Possible Implementation:
Add a way to make WHs always detonate as soon as they touch the building, instead of going straight to the center
```ini
[SomeProjectile]
or
[SomeWarhead]
IgnoreBuildingCenter=yes
immediatelydetonate=yes
Cell.Calculation=yes

or add a universal tag for Projectiles to always detonate their WHs as soon as they touch the building and of course make it optional
```

**Comment by mevitar** [comment]
Ares already has SubjectToBuildings=yes, which makes the projectile detonate when it hits the building's box.

**Comment by CommanderNour** [comment]
sorry didn't know about that tag

**Comment by CommanderNour** [comment]
After testing Ares tag it seems that it doesn't do exactly what I wanted
